ee-security/src/main/java/org/wildfly/extension/eesecurity/EESecurityDependencyProcessor.java

```diff
@@ -40,7 +40,7 @@ public void deploy(DeploymentPhaseContext phaseContext) throws DeploymentUnitPro
         final ModuleLoader moduleLoader = Module.getBootModuleLoader();
 
         final ModuleSpecification moduleSpec = unit.getAttachment(Attachments.MODULE_SPECIFICATION);
-        moduleSpec.addSystemDependency(new ModuleDependency(moduleLoader, ModuleIdentifier.fromString("javax.security.enterprise.api"), false, false, true, false));
+        moduleSpec.addSystemDependency(new ModuleDependency(moduleLoader, ModuleIdentifier.fromString("jakarta.security.enterprise.api"), false, false, true, false));
```

Comment:
Please leave the module name unchanged. This module provides classes in the javax.security.enterprise package. It isn't necessary to change the module name.

Reply:
Thank you. Fixed.